docs/3_finetuner_in_action/3_2_image_to_image.md

```diff
@@ -17,30 +17,32 @@ The dataset consists of pairs of images, these are the positive pairs. Negative
 After fine-tuning, the embeddings of positive pairs are expected to be pulled closer, while the embeddings for negative pairs are expected to be pushed away.
 
 
-## Preparing data
-Training and evaluation data are already prepared and pushed to Hubble following the {ref}`instructions <create-training-data>`.
-You can either pull the data:
-```python
-from docarray import DocumentArray
-train_data = DocumentArray.pull('resnet-ttl-train-data')
-eval_data = DocumentArray.pull('resnet-ttl-eval-data')
+## Data
+Out journey starts locally. We have to {ref}`prepare the data and push it to the cloud <create-training-data>` and Finetuner will be able to get the data by its name. For this example,
+we already prepared the data, and we'll provide the names of training and evaluation data (`resnet-ttl-train-data` and `resnet-ttl-eval-data`) directly to Finetuner.
+
+```{admonition} 
+:class: tip
+We don't require to push data to the cloud by yourself. Instead of a name, you can provide a `DocumentArray` and Finetuner will do the job for you.
 ```
-Or specify given `DocumentArray` names (`resnet-ttl-train-data` and `resnet-ttl-eval-data`) directly to Finetuner.
 
-## Choosing the model
-Now let's see what backbone models we can use. You can see available models either in {ref}`choose backbone <choose-backbone>` section or by calling `finetuner.describe_models()`.
+
+## Backbone model
+Now let's see which backbone models we can use. You can see available models either in {ref}`choose backbone <choose-backbone>` section or by calling [`finetuner.describe_models()`](../../api/finetuner/#finetuner.describe_models).
```

Comment:
I think it's easier to link to the developer reference like this: {meth}`~finetuner.describe_models()`